### PR TITLE
February 2020 test-fix ready for QC and review

### PR DESCRIPTION
**Pull request by kylebunting**
The February 2020 Migrating SQL databases to Azure test-fix is ready for QC and review. 

I included a few minor enhancements in this test-fix, to include some of the database configurations and start solution download into scripts executed by the ARM template.

### CODE REVIEW

File: Hands-on lab/HOL step-by-step - Migrating SQL databases to Azure.md [hunk 1]
```diff
@@ -57,9 +57,6 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/in
   - [Exercise 6: Enable Dynamic Data Masking](#exercise-6-enable-dynamic-data-masking)
     - [Task 1: Enable DDM on credit card numbers](#task-1-enable-ddm-on-credit-card-numbers)
     - [Task 2: Apply DDM to email addresses](#task-2-apply-ddm-to-email-addresses)
-  - [Exercise 7: Use online secondary for read-only queries](#exercise-7-use-online-secondary-for-read-only-queries)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Migrating SQL databases to Azure.md]: Hello!  Exercise 7 has been removed from the TOC but there's still an Exercise 7 in the body of the lab.  Please review and advise.

Comment (DawnmarieDesJardins) [reply Hands-on lab/HOL step-by-step - Migrating SQL databases to Azure.md]: @kylebunting @timahenning 
Hello!  Not sure if you automatically get notified I asked a question or no...so I'm adding an @ for you. :)

Comment (timahenning) [reply Hands-on lab/HOL step-by-step - Migrating SQL databases to Azure.md]: Carey is looking into to it and should have it resolved by today.

Comment (codingbandit) [reply Hands-on lab/HOL step-by-step - Migrating SQL databases to Azure.md]: All fixed @DawnmarieDesJardins

**Pull request merged**